Sources/SyrupCore/Extensions/String+Extensions.swift

```diff
@@ -22,6 +22,7 @@
  * THE SOFTWARE.
  */
 import Foundation
+import CommonCrypto
```

Comment:
Not for now but maybe we can look into switching to swift crypto later if it provides nicer APIs: https://swift.org/blog/crypto/